packages/runtime/container-runtime/src/summary/documentSchema.ts

```diff
@@ -311,15 +363,20 @@ function and(
 			desiredDocSchema.runtime[key],
 		);
 	}
+
+	const minVersionForCollab =
```

Comment:
Add a comment explaining the logic, about how for the intersection of two schemas, we keep the persisted one for this property, if present.  Even if the desired one is lower (it only moves forward).